bundles/org.openhab.binding.freeboxos/src/main/resources/freeboxECCRootCA.crt

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Just wondering about this cert and if you can tell us more about it, mainly... Is it public knowledge that the company does not mind having published? Did it come from reverse engineering?

Reply:
I'll dig a bit more on this. It is the way it is currently done in the Freebox binding.

Reply:
IIRW it is because the certificate used by the Freebox  server is not trusted. So if we do not provide it to the JVM, the connection can not be established in https.